src/gui/pedalboard/cables/Cable.cpp

```diff
@@ -102,7 +102,13 @@ void Cable::paint (Graphics& g)
 
         endPortLocation = CableViewPortLocationHelper::getPortLocation ({ endEditor, connectionInfo.endPort, true }).toFloat();
         endColour = endEditor->getColour();
-        levelDB = connectionInfo.endProc->getInputLevelDB (connectionInfo.endPort);
+        auto updatedLevelDb = jlimit (floorDB, 0.0f, connectionInfo.endProc->getInputLevelDB (connectionInfo.endPort));
+        auto levelDifference = std::abs (updatedLevelDb) - std::abs (levelDB);
+        if (std::abs (levelDifference) > 1.0f)
+        {
+            levelDB = updatedLevelDb;
+            repaint();
+        }
```

Comment:
Only update repaint cable when level is changed

Reply:
Yeah... do you think it would work to move this code to some function like `checkNeedsRepaint()` or something, and then that could get called from `CableView::timerCallback()`?

Reply:
Yeah good idea I'll look into that